src/index.js

```diff
@@ -5,20 +5,21 @@ import $ from "jquery"
 import Popper from "popper.js"
 import "bootstrap-material-design"
 import tippy from "tippy.js"
+import "offcanvas-bootstrap/dist/js/bootstrap.offcanvas.js"
 
 window.jQuery = $
 window.$ = $
 window.Popper = Popper
 
 $(document).ready(() => {
   $("body").bootstrapMaterialDesign()
```

Comment:
Should we remove bootstrap-material-design as a dependency at this point?

Reply:
test comment

Reply:
I'm planning to get rid of it soon, right now it is still used for the search bar at the top, so removing it I ran into some issues.